assets/results.js

```diff
@@ -1,84 +1,65 @@
+var ingredientUrl = "https://www.thecocktaildb.com/api/json/v1/1/search.php?i="
+var nameUrl = "https://www.thecocktaildb.com/api/json/v1/1/search.php?s="
+var randomUrl = "https://www.thecocktaildb.com/api/json/v1/1/random.php"
```

Comment:
Not quite what I meant but still like them being in one place more. Was thinking something like:

var baseCocktailUrl = "https://www.thecocktaildb.com/api/json/v1/1/"

And then we could just add the ends. But good job!